src/Orders/EmailAccounts/Responses/CreateResponse.php

```diff
@@ -33,6 +34,11 @@ public function __construct(array $attributes = [])
     {
         parent::__construct($attributes);
 
+        // Not all responses contain a status it would seem, for example get business email.
+        if (!$this->wasSuccessful()) {
+            throw new ResponseException($this->response['message']);
+        }
+
         if (!array_key_exists('user', $this->response)) {
```

Comment:
Space after “not” operator.

Reply:
We haven't done that styling anywhere else within the package though.